netlify-plugin-cloudinary/src/lib/cloudinary.ts

```diff
@@ -186,20 +184,31 @@ export async function getCloudinaryUrl(options: CloudinaryOptions) {
     if (canSignUpload) {
       // We need an API Key and Secret to use signed uploading
 
-      results = await cloudinary.uploader.upload(fullPath, {
-        ...uploadOptions,
-      })
+      try {
+
+        results = await cloudinary.uploader.upload(fullPath, {
+          ...uploadOptions,
+        })
+      } catch (error) {
+        console.error(ERROR_ASSET_UPLOAD, error)
+        throw Error(ERROR_ASSET_UPLOAD)
+      }
     } else {
       // If we want to avoid signing our uploads, we don't need our API Key and Secret,
       // however, we need to provide an uploadPreset
-
-      results = await cloudinary.uploader.unsigned_upload(
-        fullPath,
-        uploadPreset,
-        {
-          ...uploadOptions,
-        },
-      )
+      try {
+        console.log('Uploading')
```

Comment:
do we need this log? it may make sense to track in the logs which asset is being uploaded for better logging, but this doesnt give any useful information